src/SFA.DAS.QnA.Database/projects/epaoall/sections/section5.json

```diff
@@ -406,17 +406,18 @@
 			"PageId": "3",
 			"SequenceId": null,
 			"SectionId": null,
-			"Title": "Supporting current learners",
+			"Title": "How will you support the learners you are not going to assess?",
 			"LinkTitle": "Supporting current learners",
 			"InfoText": "",
 			"Questions": [
 				{
 					"QuestionId": "WR-03",
 					"QuestionTag": null,
-					"Label": "How will you support the learners you're not going to assess?",
+					"Label": "How will you support the learners you are not going to assess?",
 					"ShortLabel": "",
 					"InputClasses": null,
-					"QuestionBodyText": "<p class='govuk-body'>Your answer should include:</p><ul class='govuk-list govuk-list--bullet'><li>how you'll transfer these learners to other EPAOs</li><li>details of any EPAOs you've spoken to</li></ul>",
+					"QuestionBodyText": "<p class='govuk-body'>Your answer must include:</p><ul class='govuk-list govuk-list--bullet'><li>how you'll transfer these learners to other EPAOs</li><li>details of any EPAOs you've spoken to</li><li>If you have no registered learners, enter 'not applicable'.</li></ul>",
+					"\",": null,
```

Comment:
What is this bit here - "\",": null,  - where did it come from?

Reply:
Most probably from me, but not sure how, I copied the text from the content doc. Thanks for picking it up